logic/src/test/java/org/itsallcode/whiterabbit/logic/service/AppServiceTest.java

```diff
@@ -110,7 +110,7 @@ void setUp()
         appService = createAppService(appServiceCallback, workingTimeService);
         appService.setUpdateListener(updateListenerMock);
         modelFactory = new JsonModelFactory();
-        contractTerms = new ContractTermsService(configMock);
+        contractTerms = new ContractTermsService(TestingConfig.builder().build().getCurrentHoursPerDay());
```

Comment:
Directly define hoursPerDay, e.g. `Duration.ofHours(8)`. Then you don't need to create the TestingConfig at all, and it's clear which value is used.

Reply:
Very good! 
Done.